wasmsdk/cache.go

```diff
@@ -0,0 +1,46 @@
+//go:build js && wasm
+// +build js,wasm
+
+package main
+
+import (
+	"sync"
+	"time"
+
+	"github.com/0chain/gosdk/zboxcore/sdk"
+)
+
+type cachedAllocation struct {
+	Expiration time.Time
+	Allocation *sdk.Allocation
+}
+
+var (
+	cachedAllocations      = make(map[string]*cachedAllocation)
+	cachedAllocationsMutex sync.Mutex
```

Comment:
Use LRU cache as suggested in another PR

Reply:
fixed